### ISSUE TITLE
[enhancement] Add a feature to delete old torrents from previous updates.

### ISSUE THREAD

**Issue by nikolailarsen01** [ccdale/manjaro-torrent-find#3]
Add a feature to delete old torrents, so the directory does not get bloated with old files.

**Comment by ccdale** [comment]
good idea, I'll have to take a record of what files are downloaded, compare them with the list of files to download, and delete older versions - hopefully file names will be consistent.

**Comment by nikolailarsen01** [comment]
They had a forum post on the old forum about iso file naming, so I that they will stick to the current naming scheme for a while.
